src/server/pkg/deploy/cmds/cmds.go

```diff
@@ -284,3 +292,18 @@ you used the "--static-etcd-volume" flag), the underlying volume will not be
 removed.`)
 	return []*cobra.Command{deploy, undeploy}
 }
+
+func getDefaultDashImage() string {
+	// Try to get the latest according to our pachctl version, or fall back to the encoded default
+	url := fmt.Sprintf("https://pachyderm.io/versions/dash/%v.%v/latest", version.MajorVersion, version.MinorVersion)
```

Comment:
We could. But I like the convenience offered here.

Short term this is sugar to make it more convenient for us to use / test out the latest UI. The UI is still in alpha so there could definitely be breaking changes ... buts its alpha ... so I don't see that as an issue. Nor do I think that we have a short term need to support the UI on an out of date version of pachctl ... we aggressively ask users to upgrade anytime they see a bug.

Longer term (once we're out of alpha/beta), we'd probably want to use the `/stable` path in lieu of `/latest`. I agree once we're out of alpha/beta we need to agree and adhere to versioning practices that will allow a user w a given version not to accidentally install a UI that has breaking changes. The preliminary idea here was to make breaking changes per minor release (which is why we use the major/minor version in the url).